src/components/AssetsPage/pageTypes.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Button from '@edx/paragon/src/Button';
+
+import WrappedAssetsDropZone from '../AssetsDropZone';
+import WrappedAssetsTable from '../AssetsTable';
+import WrappedAssetsFilters from '../AssetsFilters';
+import WrappedPagination from '../Pagination';
+import edxBootstrap from '../../SFE.scss';
+
+export const types = {
```

Comment:
I'm not married to these enums. I'm happy to change them to something else if there are other opinions for a better naming scheme.